config/routes.rb

```diff
@@ -3,4 +3,5 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+  get '/', to: 'pages#index'
```

Comment:
Perhaps would be better to add as a root_path:

```suggestion
  root to: 'pages#index'
```

Reply:
I changed it accordingly.